src/protocol/fluvio-protocol-api/src/response.rs

```diff
@@ -68,17 +67,6 @@ where
 
         let mut src = Cursor::new(&data);
 
-        let mut size: i32 = 0;
```

Comment:
The test is passing with and without this. I will add a test for this, but first I have to check where is done the encode_to_file or something like that to check if it includes the size or not.

Reply:
it should work as is since it is not dealing with transport

Reply:
My main doubt is because in fluvio/src/protocol/src/store.rs, it looks like file_encode() adds the length first. But I don't know if that is related to this. If that is true, this change should be removed and also the change in api.rs and request.rs related to this.